### PR TITLE
symfonize statistics index

### PR DESCRIPTION
**Pull request by sillydebs**

### CODE REVIEW

File: src/Controller/StatisticController.php [hunk 1]
```diff
@@ -15,10 +15,21 @@
 include_once __DIR__ . "/../../public/conges/class.conges.php";
 include_once __DIR__ . "/../../public/include/function.php";
 require_once __DIR__ . "/../../public/include/db.php";
+require_once __DIR__ . "/../../public/statistiques/class.statistiques.php";
 include_once __DIR__ . '/../../public/absences/class.absences.php';
+include_once __DIR__ . '/../../public/postes/class.postes.php';
```
Comment (alexarn) [inline src/Controller/StatisticController.php:20]: Why did you add class statistiques and postes ? There are not used in the controller

File: templates/statistics/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  <h3>Statistiques</h3>
+  <ul>
+    <li><a href='index.php?page=statistiques/temps.php'>Feuille de temps</a></li>
+    <li><a href='index.php?page=statistiques/agents.php'>Par agent</a></li>
+    <li><a href='index.php?page=statistiques/service.php'>Par service</a></li>
+    <li><a href='index.php?page=statistiques/statut.php'>Par statut</a></li>
+    <li><a href='index.php?page=statistiques/postes.php'>Par poste</a></li>
+    <li><a href='index.php?page=statistiques/postes_synthese.php'>Par poste (Synthèse)</a></li>
+    <li><a href='index.php?page=statistiques/postes_renfort.php'>Postes de renfort</a></li>
```
Comment (alexarn) [inline templates/statistics/index.html.twig:15]: Please use asset in href

**Review by alexarn — CHANGES_REQUESTED** [review]

**Comment by alexarn** [comment]
File public/statistiques/index.php must be deleted

File: templates/statistics/index.html.twig (hunk 2 above)
Comment (sillydebs) [reply templates/statistics/index.html.twig:15]: Should I use asset in href despite the absence of some pages for now ?

Comment (sillydebs) [reply templates/statistics/index.html.twig:15]: Several pages are symfonized in PR after this current one. Changing links will lead to errors while the new pages aren't merged in master....

File: src/Controller/StatisticController.php (hunk 1 above)
Comment (sillydebs) [reply src/Controller/StatisticController.php:20]: Both classes are used in other methods of the controller. I forgot to delete them temporarily, my bad.

File: templates/statistics/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+  <h3>Statistiques</h3>
+  <ul>
+    <li><a href='{{ asset("statistics/time") }}'>Feuille de temps</a></li>
+    <li><a href='{{ asset("statistics/agent") }}'>Par agent</a></li>
+    <li><a href='{{ asset("statistics/service") }}'>Par service</a></li>
+    <li><a href='{{ asset("statistics/status") }}'>Par statut</a></li>
+    <li><a href='{{ asset("statistics/position") }}'>Par poste</a></li>
+    <li><a href='{{ asset("statistics/positionsummary") }}'>Par poste (Synthèse)</a></li>
+    <li><a href='{{ asset("statistics/supportposition") }}'>Postes de renfort</a></li>
```
Comment (alexarn) [inline templates/statistics/index.html.twig:15]: You should not set href with routes that don't exist yet. When i said "use asset", i mean with old url. This way:
<li><a href='{{ asset("index.php?page=statistiques/temps.php") }}'>Feuille de temps</a></li>

**Review by alexarn — CHANGES_REQUESTED** [review]

File: templates/statistics/index.html.twig (hunk 3 above)
Comment (alexarn) [reply templates/statistics/index.html.twig:15]: asset("index.php?page=statistiques/temps.php")

File: public/setup/atomicupdate/update_path.php [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?php
+
+$sql[]="UPDATE `{$dbprefix}acces` SET `page`='/statistics' WHERE `page` = 'statistiques/index.php';";
```
Comment (alexarn) [inline public/setup/atomicupdate/update_path.php:3]: You can just remove the page (not the entry) and provide the permissions in config/permissions.yaml

File: public/setup/db_data.php [hunk 5]
```diff
@@ -39,7 +39,7 @@
...
 $sql[]="INSERT INTO `{$dbprefix}acces` (`nom`,`groupe_id`,`groupe`,`page`,`categorie`,`ordre`) VALUES ('Planning Poste - Importer un modèle', 301, 'Création / modification des plannings, utilisation et gestion des modèles', 'planning/poste/importer.php','Planning','110');";
 $sql[]="INSERT INTO `{$dbprefix}acces` (`nom`,`groupe_id`,`groupe`,`page`,`categorie`,`ordre`) VALUES ('Planning Poste - Enregistrer un modèle', 301, 'Création / modification des plannings, utilisation et gestion des modèles', 'planning/poste/enregistrer.php','Planning','110');";
-$sql[]="INSERT INTO `{$dbprefix}acces` (`nom`,`groupe_id`,`groupe`,`page`,`categorie`,`ordre`) VALUES ('Statistiques', 17, 'Accès aux statistiques', 'statistiques/index.php','Statistiques','170');";
+$sql[]="INSERT INTO `{$dbprefix}acces` (`nom`,`groupe_id`,`groupe`,`page`,`categorie`,`ordre`) VALUES ('Statistiques', 17, 'Accès aux statistiques', '/statistics','Statistiques','170');";
```
Comment (alexarn) [inline public/setup/db_data.php:42]: You can just remove the page (not the entry) and provide the permissions in config/permissions.yaml

**Review by alexarn — CHANGES_REQUESTED** [review]
Why is there changes on the file templates/statistics/attendeesmissing.html.twig ?

**Review by alexarn — APPROVED** [review]

**Comment by alexarn** [comment]
Merged on master
